### PR TITLE
ui: add hardware metrics to new "Hardware" dashboard

### PR DESCRIPTION
**Pull request by vilterp**
First two commits are from #27137; that PR will go in first.

image

TODO
- [ ] tooltips
- [ ] format CPU percentage (currently 100% of one core = 1)

### CODE REVIEW

File: pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/hardware.tsx [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+export default function (props: GraphDashboardProps) {
+  const { nodeIDs, nodesSummary, nodeSources, tooltipSelection } = props;
+
+  return [
+    <LineGraph
+      title="CPU Percent"
+      sources={nodeSources}
+    >
+      {/* TODO(vilterp): multiply by 100 to get percent */}
+      <Axis units={AxisUnits.Count} label="CPU">
```
Comment (couchand) [inline pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/hardware.tsx]: Oh, hey, at one point I removed an AxisUnits.Percent cause it wasn't used anywhere...  Here ya go: https://github.com/cockroachdb/cockroach/commit/70c44194d917ef9f9463c9d18b8ed29079f5faaa

File: pkg/ui/src/views/cluster/containers/nodeGraphs/index.tsx [hunk 2]
```diff
@@ -48,7 +49,8 @@ const dashboards: {[key: string]: GraphDashboard} = {
   "replication": { label: "Replication", component: replicationDashboard },
   "distributed": { label: "Distributed", component: distributedDashboard },
   "queues": { label: "Queues", component: queuesDashboard },
-  "requests": { label: "Slow Requests", component: requestsDashboard},
+  "requests": { label: "Slow Requests", component: requestsDashboard },
+  "hardware": { label: "Hardware", component: hardwareDashboard },
```
Comment (couchand) [inline pkg/ui/src/views/cluster/containers/nodeGraphs/index.tsx]: I would sort it much higher, maybe second after "overview"? @piyush-singh @josueeee @Amruta-Ranade thoughts?

**Comment by couchand** [comment]
LGTM

File: pkg/ui/src/views/cluster/containers/nodeGraphs/index.tsx (hunk 2 above)
Comment (vilterp) [reply pkg/ui/src/views/cluster/containers/nodeGraphs/index.tsx]: Done

File: pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/hardware.tsx (hunk 1 above)
Comment (vilterp) [reply pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/hardware.tsx]: Done

**Pull request merged**